geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/io/LayerReaderFactory.scala

```diff
@@ -50,6 +50,12 @@ abstract class LayerReaderWrapper {
     projQuery: String,
     numPartitions: Int
   ): TiledRasterRDD[_]
+
+  def readMetadata(
+    keyType: String,
+    layerName: String,
+    zoom: Int
+  ): String
```

Comment:
This seems like it belongs on `AttributeStoreWrapper` unless there is super compelling reason for it to be here?